test/client/before-test.js

```diff
@@ -1,4 +1,7 @@
 (function () {
+    $('script').addClass('script-TC2b9a6d');
+    $('link').addClass('ui-stylesheet-TC2b9a6d');
```

Comment:
We 'hide' all of the 'service' scripts/css on the page before hammerhead initialized, otherwise hammerhead process `qunit-harness` resources (qunit, jquery etc). I think this is correct, because there are no other scripts on the page in this moment (scripts that we should not 'shadow').

We can specify the scripts and links but we will have to correct that  when some changes in the qunit-harness implemented